website/docs/en/plugins/list/index.mdx

```diff
@@ -66,6 +66,7 @@ The following are common framework-agnostic plugins:
 - [Stylus Plugin](/plugins/list/plugin-stylus): Use Stylus as the CSS preprocessor.
 - [Check Syntax Plugin](/plugins/list/plugin-check-syntax): Used to analyze the syntax compatibility of artifacts, to see if there are any advanced syntaxes that may cause compatibility issues.
 - [CSS Minimizer Plugin](/plugins/list/plugin-css-minimizer): Used to customize CSS minimizer, switch to [cssnano](https://cssnano.co/) or other tools for CSS compression.
+- [Typed CSS Modules Plugin](/plugins/list/plugin-typed-css-modules): Used to generate TypeScript declaration file for CSS modules.
```

Comment:
Please update the packages list too:

https://rsbuild.dev/guide/start/npm-packages